### DISCUSSION TITLE
Nom de domaine pour le site

### DISCUSSION THREAD

**Discussion by florinesueur** [cnumr/best-practices-wordpress#43] [General] [answered]
Je viens seulement d'y penser.
Faut-il acheter un nom de domaine ou est-il possible d'avoir un sous domaine du CNUMR ou de l'asso ?
Je peux demander à l'asso, si nécessaire.

**Comment by dNicolle** [comment]
Effectivement, si tu peux demander à l'asso c'est cool.
Et ptet anticiper aussi le fait qu'on puisse faire la même chose plus tard avec les 115 BP web dans la demande, pour choisir l'alias ou le ndd qui permettra de continuer des déclinaisons si d'autres veulent en faire ?

**Comment by florinesueur** [comment]
La réponse de l'asso.

Pour le logo : Asso + cnumr dans le doc.

Pour le sous domaine : greenit.eco qui va porter toute la partie associative
, on le maitrise donc on peut faire un SD en 2/2


avec possibilité d'hébergement

**Reply by hrenaud** [reply]
Pour l'hébergement, il faudra voir si ça peut build du Gatsby.

GitHub peut le faire gratuitement, on peut faire une redirection DNS pour pointer dessus.

---

J'ai ajouté le logo sur le readme et sur le site.

**Reply by florinesueur** [reply]
Je suis ok aussi pour la solution Github.

**Comment by florinesueur** [comment]
Hello,

Quel sous-domaine greenit.eco vous souhaitez ? Une idée ?

**Reply by hrenaud** [reply]
Il faut le réfléchir globalement, il y a deux référentiels, les 115 bp et les bp WordPress.

On peut imaginer deux sites, un pour chacun ou fusionner les deux référentiels. Pour le moment, l'idée de fusionner n'a pas été abordé, donc on reste dans l'idée d'avoir deux urls distincts, mais anticipons des noms en cohérence.

- bp.geenit.eco ou 115bp.geenit.eco ou bp-generiques.greenit.eco ou etc.
- bp-wordpress.geenit.eco

Qu'en pensez-vous ?

**Reply by dNicolle** [reply]
Completement en phase avec un NDD générique pour le referentiel des 115BP d'écoconception web, et un alias qui en découle pour les référentiels dédiés.

**Reply by florinesueur** [reply]
Il faudrait voir ça avec Fred et l'équipe des 115BP.
A mon sens, le mot referentiel est plus parlant et plus officiel que bp en lisant Dominique.

Je pense à quelque chose comme :
- referentiel-web.greenit.eco
- referentiel-wordpress.greenit.eco

Il pourrait y avoir par la suite : 
- referentiel-mobile.greenit.eco
- referentiel-iot.greenit.eco
- referentiel-shopify.greenit.eco
- ...

Qui peut se charger de prendre la température auprès des autres pour partir du bon pied directement ?

**Reply by hrenaud** [reply]
Ça fait long à saisir dans son navigateur quand on ne l'a pas (encore) en raccourci.
Mais c'est franchement plus parlant !

**Comment by bordage** [comment, marked as answer]
Bonjour,

Et pourquoi pas : 
ref.greenit.eco/mobile
ref.greenit.eco/iot
ref.greenit.eco/shopify
ref.greenit.eco/WP
...

cela aurait deux avantages : 
1. cela renforce le SEO sur un seul domaine
2. c'est plus facile à gérer (ajout d'un simple répertoire).

**Comment by hrenaud** [comment]
@bordage Tu sais quoi mettre à la racine ref.greenit.eco ?
